docs/services/topology.md

```diff
@@ -154,7 +157,7 @@ These instructions assume that the code has not been merged to master.
 
 1.  Test the web interface at <https://topology-itb.opensciencegrid.org>.
 
-    Errors and output are in `/var/log/httpd/error_log` (for both production and ITB).
+    Errors and output are in `/var/log/httpd/error_log`.
```

Comment:
How do we determine what messages correspond to production vs ITB?

Reply:
They're on different hosts now.

Reply:
Oh, duh